code/src/App.js

```diff
@@ -1,9 +1,42 @@
-import React from 'react'
+import React, { useState }  from 'react'
+import { NameQuestion }  from './Components/NameQuestion.js';
+import { NoseQuestion }  from './Components/NoseQuestion.js';
+import { EyeQuestion }  from './Components/EyeQuestion.js';
+import { SmileQuestion }  from './Components/SmileQuestion.js';
+import { MustacheQuestion }  from './Components/MustacheQuestion.js';
+import { Heading }  from './Components/Heading.js';
+import { SubmitQuestions }  from './Components/SubmitQuestions.js';
+import { Respons }  from './Components/Respons.js';
+
 
 export const App = () => {
+const [name, setName] =useState();
+
+const [sizeOfNose, setSizeOfNose] = useState("")
+
+const [eyeColor, setEyeColor] = useState("");
+
+const [smile, setSmile] = useState()
```

Comment:
Should this also have a default empty string as the other face-states? I suppose they should all have the same default value though.

Reply:
it would be nice, but not necessary here 🦄